src/types/widget.ts

```diff
@@ -89,6 +91,12 @@ export interface WidgetStateData {
    */
   response: SentinelResponse | string;
 
+  /**
+   * The WidgetMode returned from the API. Either "interactive" or "noninteractive". Smart mode ("automatic")
```

Comment:
Perhaps the explanation can say something like the following about Smart mode:
> Smart Mode ("automatic") automatically chooses between "interactive" and "noninteractive". 

And maybe "The widget mode is configured in the Friendly Captch dashboard" 🤔